types/artifacts.d.ts

```diff
@@ -17,11 +17,11 @@ type _TaskNode = import('../lighthouse-core/lib/tracehouse/main-thread-tasks.js'
 import AuditDetails from './lhr/audit-details'
 import Config from './config';
 import Gatherer from './gatherer';
-import {IcuMessage} from './lhr/i18n';
+import { IcuMessage } from './lhr/i18n';
```

Comment:
would you revert all of these formatting changes please? makes it hard to tell what the underlying change is :)

Reply:
My IDE automatically converted these I will change them back!

Reply:
I was not able to revert it. But I deleted that file and push another commit with the required change only.